.gitmodules

```diff
@@ -1,5 +1,6 @@
 [submodule "rln/vendor/rln"]
 	path = rln/vendor/rln
+	ignore = dirty
```

Comment:
Did you make any relevant changes here?

Reply:
Not really, I was annoyed by dirty submodule changes, and I saw that for semaphore there was this extra flag so I added it here and dirty submodules changes are now ignored.